examples/logs-slack-alerts/variables.tf

```diff
@@ -0,0 +1,13 @@
+variable "cron_schedule" {
+  description = "Cron schedule for Google Cloud Scheduler"
+  default = "55 * * * *"
+}
+
+variable "loader_alert_cloud_function_bucket"{
+  description = "GCS bucket where python script + requirements.txt is located"
+}
+
+variable "loader_alert_cloud_function_zip"{
```

Comment:
If you use `terraform-google-scheduled-function` the zipping and uploading is all done for you

Reply:
Tested and works awesome